pkg/apis/ignite/v1alpha1/conversion.go

```diff
@@ -174,3 +174,10 @@ func Convert_v1alpha1_VMKernelSpec_To_ignite_VMKernelSpec(in *VMKernelSpec, out
 
 	return Convert_v1alpha1_OCIClaim_To_ignite_OCI(&in.OCIClaim, &out.OCI)
 }
+
+// Convert_v1alpha1_VMNetworkSpec_To_ignite_VMNetworkSpec calls the autogenerated conversion function and possibly custom conversion logic
+func Convert_v1alpha1_VMNetworkSpec_To_ignite_VMNetworkSpec(in *VMNetworkSpec, out *ignite.VMNetworkSpec, s conversion.Scope) error {
+	// .Spec.Network.Mode has been removed in v1alpha2. However, we won't persist that information anywhere in the new API, so
```

Comment:
`.Spec.Network.Mode` -> `.spec.network.mode`, the wording on this comment is also a bit funky. Just something like "The network mode is not tracked in the new API, so there's no extra conversion logic" would be enough.